.dockerignore

```
@@ -0,0 +1,4 @@
+_posts/
+assets/
+_docs/
+_includes/
```

```suggestion
/.git/
/_site/
/_posts/
/assets/
/_docs/
/_includes/
/.idea/
/*.iml
/README.md
```

1. Surprisingly, the `.git` directory is needed inside the container, but it adds extra 400Mb to the image size. But we can add `repository: intellij-rust/intellij-rust.github.io` line to the `_config.yml` and so the `.git` directory will not be needed
2. `_site` is a kind of a cache, it's better to remove it from the image and mount it instead
3. `.idea`, `*.iml` and  `README.md` are not needed as well

---

Just remembered that `/.run/` should be ignored as well